fdbclient/BlobGranuleFiles.cpp

```diff
@@ -289,7 +291,9 @@ struct IndexBlockRef {
 		                                     eKeys.headerCipherKey,
 		                                     cipherKeysCtx.ivRef.begin(),
 		                                     AES_256_IV_LENGTH,
-		                                     ENCRYPT_HEADER_AUTH_TOKEN_MODE_SINGLE,
+		                                     FLOW_KNOBS->ENCRYPT_HEADER_AUTH_TOKEN_ENABLED
```

Comment:
we can hide the knob check in BlobCipher code?

Reply:
Absolutely, we can abstract out this code in either EncryptUtils Or BlobCipher.
Will make the necessary changes.